src/gui/tray/ActivityItem.qml

```diff
@@ -21,7 +21,9 @@ MouseArea {
     enabled: (model.path !== "" || model.link !== "" || model.isCurrentUserFileActivity === true)
     hoverEnabled: true
 
-    height: childrenRect.height
+    // We add some height to the height of the children elements as a way of adding padding.
+    // We center the children vertically in the middle of this MouseArea.
+    height: childrenRect.height + 20
```

Comment:
better use a named constant for `20`
hopefully the need for a comment may be removed by using a named constant here

Reply:
Good idea, created new constant